lighthouse-core/report/html/report-styles.css

```diff
@@ -1222,7 +1222,7 @@
 
 .lh-categories {
   width: 100%;
-  overflow: hidden;
+  /* overflow: hidden; */
```

Comment:
?

Reply:
agreed with the above :) delete or put back?